README.md

```diff
@@ -60,7 +60,7 @@ npm start
 - [electron/electron-api-demos](https://github.com/electron/electron-api-demos) - an Electron app that teaches you how to use Electron
 - [hokein/electron-sample-apps](https://github.com/hokein/electron-sample-apps) - small demo apps for the various Electron APIs
 
-## Programmatic usage
+## Programmatic Usage
```

Comment:
The existing capitalization is actually consistent with our styleguide: https://github.com/electron/electron/blob/master/docs/styleguide.md#titles